client/src/components/screens/Patient/components/FilesTab/index.tsx

```diff
@@ -26,7 +26,7 @@ const FilesTab = () => {
   const patient = useSelector((state: State) => state.patient.patient.parsed) as ParsedPatientData;
   const [isOpen, setIsOpenModal] = useState<boolean>(false);
   const [documentReference, setDocumentReference] = useState<string>('');
-  let filesResults = getFilesData({ patientId: patient.id })();
+  let filesResults = getFilesData({ patientId: patient.id });
```

Comment:
I don't think let is necessary. const would do the job

Reply:
Oui je réassigne a la ligne 33